libs/vm-modules/src/math/tensor/tensor.cpp

```diff
@@ -75,7 +75,8 @@ void VMTensor::Bind(Module &module, bool const enable_experimental)
   using Index = fetch::math::SizeType;
 
   auto tensor_constructor_charge_estimate = [](Ptr<Array<SizeType>> const &shape) -> ChargeAmount {
-    SizeType padded_size = fetch::math::Tensor<DataType>::PaddedSizeFromShape(shape->elements);
+    DataType padded_size =
```

Comment:
i don't really understand why this change is made? It seems to result in additional casting in the tensor estimator and is anyway a confusing representation of padded_size (size has to be integral anyway)